tests/regression_tests.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 14 16:17:57 2020
+
+@author: smullally
+"""
+
+
+import exovetter as vet
+
+
+def test_lpp():
+    """
+    Test reads in an lpp file name, runs the metric without \
+    any plots and returns values.  
+    It then asseerts that the normal one is greater than 0, which must be true.
+    """
+    
+    tce_filename = "tce0001234567_dvt.fits"
+    
+    #Populate the tce class by reading in a file
+    #At this point a phases array, 
+    #and (int) transit_number would get populated.
+   
+    tce = vet.create_tce(file = tce_filename, ext = 1)
+    
+    #This populates the LPP metric fields in this class
+    #It also populates where the plotting data is stored
+    #This function relies on inputs from a precomputed model stored
+    #in something called mapInfo. In theory if someone trains up another 
+    #model we could input it here.
+    tce.run_lpp(mapInfo = None)
+    
+    norm_TLpp = tce.lpp.norm
+    raw_TLpp = tce.lpp.raw
+    binned_lpp_transit = tce.lpp.binned_transit
```

Comment:
This suggests that maybe `lpp` is actually another class; i.e., `tce.run_lpp` would return/create some `LPPClass` with attributes like `norm`, `raw`, etc populated?

Reply:
Yes, I think you are right here.  But it would be such a simple class..  It could just return the values, but I want them kept some where, so they can then be used to write them out or used by other methods.
It could be 
`tce.run_lpp`
`tce.lpp_norm`
`tce.lpp_raw`
`tce.lpp_binnedTransit`
Thoughts?

Reply:
What about `tce.lpp(lpp_type='raw')`? `lpp_type` will be "norm", "raw", or "binned_transit".

Reply:
It is a minor step between calculating raw and norm. You would never calculate norm without first getting raw. But it is a small amount of effort to calculate norm once you have raw.  The binned transit is produced before either and is an array. The other two are floats.

this kind of thing is going to be true of many of these metrics. There is one function that produces several values after doing a somewhat expensive operation.   So this will be generalized to more operations.  Another suggestion is to have a dataframe or dictionary to hold on to these values.

Reply:
If people will always want all three, then dictionary makes sense. Otherwise, given raw is pre-requisite to norm and some calculation is expensive, looks like `lpp` will need to be a class.